Documentation/internals/security-identities.rst

```diff
@@ -0,0 +1,48 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _security_identities:
+
+*******************
+Security Identities
+*******************
+
+Security identities are generated from labels. They are stored as ``uint16``,
```

Comment:
Where is the security identity stored as a `uint16`?

Reply:
For reference, here's the golang definition:

https://github.com/cilium/cilium/blob/62adcc5c24887af808741739491fcecfc6995a42/pkg/identity/numericidentity.go#L405-L411

Reply:
(given that this is internal documentation, I wonder whether we could link to this somehow from the docs. Not sure how to make those links stable against refactoring / code changes though)

Reply:
Thanks, clearly I didn't proof read this before submitting :sweat_smile: 

Good point on linking the internal docs, I think we can point at https://pkg.go.dev/github.com/cilium/cilium/pkg/identity#NumericIdentity. I will add that